code/game/objects/items/cigs_lighters.dm

```diff
@@ -101,6 +101,7 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	name = "firebrand"
 	desc = "An unlit firebrand. It makes you wonder why it's not just called a stick."
 	smoketime = 20 //40 seconds
+	custom_materials = list(/datum/material/wood = 2000)

```

Comment:
can you base all these values off the MINERAL_MATERIAL_AMOUNT? that sounds better than magic numbering something that's related to a define.